packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CryptoSelection/CryptoSelector/CryptoItem/template.success.tsx

```diff
@@ -66,7 +87,26 @@ const Success: React.FC<Props> = props => {
       {props.onClick && (
         <Icon name='chevron-right' size='32px' color='grey600' />
       )}
-    </DisplayContainer>
+      {!props.cryptoSelection && (
```

Comment:
You could just use onClick instead of passing in another property.

```
props.onClick ? <Icon name='chevron-right' size='32px' color='grey600' /> : <Icon name='chevron-right' size='32px' color='grey600' /> ...
```